node/node.go

```diff
@@ -472,23 +472,23 @@ func (n *Node) Container() *skyobject.Container {
 // yet. Use methods of this Pool to manipulate
 // connections: Dial, Connection, Connections,
 // Address, etc
-func (s *Node) Pool() *gnet.Pool {
-	return s.pool
+func (n *Node) Pool() *gnet.Pool {
```

Comment:
It called 's' everywhere by historical reasons. The Node was Server.

Reply:
It's just historical note, nothing more.

Reply:
lol, I think may be you will not let that happens in your code.